app/models/user.rb

```diff
@@ -171,7 +175,8 @@ def self.lookup_by_identity(username)
   end
 
   def self.authorize_user(userid)
-    return if userid.blank? || admin?(userid)
+    # Don't authorize users with userid == "admin"
+    return if userid.blank? || userid == "admin"
```

Comment:
Few things:

* At the very least, can we `.freeze` this string?  Or use an existing constant?
* Is there a reason this doesn't go through `MiqUserRole` or `MiqProductFeature`?  I guess for simplicity and reducing risk? (cool with that being the case, just making sure)